test/langtools/tools/javac/versions/Versions.java

```diff
@@ -403,6 +403,14 @@ public static void main(String... args) {
                  }
              }
              """),
+
+         SOURCE_25(25, "New25.java",
+             // New feature in 25: module import declarations
+             """
+             import module java.base;
+             public class New25 {
```

Comment:
Should we make this a simple source file?

Reply:
> Should we make this a simple source file?

That isn't necessary.

The purpose of this Versions.java test is focused on making sure setting --release has some effect for each release where a different source language is recognized. It is not attempting to be exhaustive that each new-in-release-N feature is accepted/rejected at appropriate source levels.

Per `javac` regression test conventions, the tests associated with a given feature should include negative tests that a new language construct is rejected on at least one older source/release version.

This test is more of a backstop insurance policy.